Makefile

```diff
@@ -40,7 +41,8 @@ generate: $(SOURCES)
 
 .PHONY: generate-deps
 generate-deps:
-	go get github.com/tools/godep
+	DEP_RELEASE_TAG=$DEP_RELEASE_TAG
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
i don't think `curl ... | sh` is something we want to be doing in a make target.